### ISSUE TITLE
[FR] Provide last processed image via HTTP

### ISSUE THREAD

**Issue by mruettgers** [roflcoopter/viseron#111]
In addition to #23 it would be great if you could add another HTTP endpoint for grabbing the last processed camera image.
I have some PIR sensors in place and I would like to send a Telegram message with a snapshot attached in case someone is moving in their detection areas.
Decoding the multipart stream is not that easy in NodeRED (which I use for doing the logic).
In the meantime I will try to grab the image sent via MQTT.
Maybe this could be an alternative for my feature request.

**Comment by roflcoopter** [comment]
Yes this wont be a problem to implement now that a basic web api is implemented, will add in a future release
